javaparser-core/src/main/javacc/java_1_8.jj

```diff
@@ -209,7 +209,7 @@ final class ASTParser {
     }
 
     private static class RangedList<T extends Node> {
-        Range range = Range.UNKNOWN;
+        Range range = Range.range(0, 0, 0, 0);
```

Comment:
I do not like that because it is not true: the element has not that position. If this value represent `Range.UNKNOWN` then we should keep `Range.UNKNOWN`